### ISSUE TITLE
🐛 [Bug]: vue component official website popover prompt box bug

### ISSUE THREAD

**Issue by Sube-py** [opentiny/tiny-vue#520]
### Version

latest

### Vue Version

3.3.4

### Link to minimal reproduction

https://opentiny.design/tiny-vue/zh-CN/os-theme/components/popover

### Step to reproduce

1. 打开vue组件的官网
2. 点击popover提示框不滑动屏幕
3. 点击一个例子的展开代码
4. 可见popover挡住了代码块
5. 同理可证，将popover用在项目中默认弹出也会出现这种问题
6. 滑动屏幕，popover复位
<img width="1512" alt="image" src="https://github.com/opentiny/tiny-vue/assets/53138073/868b2343-371c-43fe-812a-e7796b4b2c22">

### What is expected

期望popover定位正常

### What is actually happening

popover挡住其它div

### Any additional comments (optional)

复现代码：App.vue
```js
<template>
  <div class="wrapper">
    <TinyButton class="btn" @click="expandTest">展开</TinyButton>
    <div ref="testRef" class="test"></div>
    <div class="container">
      <tiny-popover placement="top-start" content="这是一段内容" trigger="manual" v-model="show">
        <template #reference>
          <tiny-button>默认</tiny-button>
        </template>
      </tiny-popover>
      <tiny-popover placement="top-start" content="这是一段内容" :arrow-offset="50" trigger="manual" v-model="show">
        <template #reference>
          <tiny-button>偏移50</tiny-button>
        </template>
      </tiny-popover>
      <tiny-popover placement="top-start" content="这是一段内容" :arrow-offset="100" trigger="manual" v-model="show">
        <template #reference>
          <tiny-button>偏移100</tiny-button>
        </template>
      </tiny-popover>
    </div>
  </div>
</template>

<script setup>
import { ref } from 'vue'
import { Popover as TinyPopover, Button as TinyButton } from '@opentiny/vue'

const show = ref(true)

const testRef = ref(null)
const isExpand = ref(false)
const expandTest = () => {
  if (!isExpand.value) {
    testRef.value.style.height = '400px';
    isExpand.value = true;
    return
  }
  isExpand.value = false;
  testRef.value.style.height = '0';
}
</script>

<style>
.wrapper {
  position: relative;
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100%;
  flex-direction: column;
  overflow: auto;
  height: 100vh;
}

.wrapper .btn {
  position: absolute;
  left: 50%;
  top: 12px;
}

.wrapper .test {
  height: 0;
  width: 100%;
  transition: 1s;
  background-color: bisque;
}

.container .reference-wrapper {
  display: block;
  margin-bottom: 80px;
}

.container .reference-wrapper button {
  width: 200px;
}
</style>
```

**Comment by gweesin** [comment]
I believe this issue falls under the category of documentation. A good solution would be to trigger a resize event when expanding or collapsing code to accommodate this scenario.

😀I'm glad to solve this issue, could you assign this issue for me?

**Comment by kagol** [comment]
Fixed by @gweesin on #535
